Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -106,7 +106,7 @@ struct DisplayHymnBottomBar: View {
                         Button(action: {
                             self.showAudioPlayer.toggle()
                         }, label: {
-                            showAudioPlayer ? Image(systemName: "play.fill").accentColor(.accentColor) : Image(systemName: "play").accentColor(.primary)
+                            (showAudioPlayer ? Image(systemName: "play.fill").accentColor(.accentColor) : Image(systemName: "play").accentColor(.primary)).padding()
```

Comment:
I think the problem isn't that it needs a `padding()`, but that it needs to be a `BottomBarLabel` not an `Image`

Reply:
Looks good. I didn't know I could put a view in a conditional modifier like that! sweet!